frost-core/src/frost.rs

```diff
@@ -44,8 +44,10 @@ where
     /// Deserializes [`BindingFactor`] from bytes.
     pub fn from_bytes(
         bytes: <<C::Group as Group>::Field as Field>::Serialization,
-    ) -> Result<Self, Error> {
-        <<C::Group as Group>::Field>::deserialize(&bytes).map(|scalar| Self(scalar))
+    ) -> Result<Self, Error<C>> {
+        <<C::Group as Group>::Field>::deserialize(&bytes)
+            .map(|scalar| Self(scalar))
+            .map_err(|e| e.into())
```

Comment:
Is this needed because the original `e` may be `FieldError` or `GroupError`?

Reply:
Exactly, in this case it's always `FieldError`, but we want an `Error`. Since `Error` has a `FieldError(#[from] FieldError),` it has a `From` implementation that allows us to just call `.into()`.